### DISCUSSION TITLE
How to replace history URL with wire:navigate?

### DISCUSSION THREAD

**Discussion by JSFTI** [livewire/livewire#7691] [1. Help]
There is a language selector in my current project. The localization uses the URL to determine localization, `{lang}/path/to/page`.

Every time user changes the site's language, the URL changes.

I used `wire:navigate` to switch the URL. However, if the browser back button is pressed, the page will return to the previous language.

Is there a way to replace the current history URL rather than push? I think the default behavior of `wire:navigate` is push.

**Comment by michaelcontento** [comment]
I also have the need for something like this 🖐️ 

Maybe implemented as an modifier? Something similar to `wire:navigate.replace=".."`? 🤔
